CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * `jj show` patches can now be suppressed with `--no-patch`.
 
+* `jj desc` adds the commit trailers, configured in the `commit_trailers` template, to the commit description. Use
+  cases include DCO Sign Off and Gerrit Change Id.
```

Comment:
nit: this file is mostly wrapped at 80 columns, so do that here too for consistency